cli/src/main/java/org/jboss/pnc/bacon/cli/App.java

```diff
@@ -117,31 +117,29 @@ public int run(String[] args) throws Exception {
                             exec.getResultHandler().onExecutionFailure(CommandResult.FAILURE, cmd);
                         }
                         throw cmd;
-                    } catch (CommandValidatorException | CommandLineParserException e) {
+                    } catch (CommandValidatorException | CommandLineParserException ex) {
```

Comment:
why e -> ex?

Reply:
Because @dwalluck wanted it.

Reply:
I think he suggested standardizing on e ;)

Reply:
Yeah true ... I can change it to e if thats what everyone wants!